main.go

```diff
@@ -87,6 +87,11 @@ func main() {
 	flag.BoolVar(&debug, "debug", false, "flag indicating whether debug output should be written")
 	flag.BoolVar(&jsonLogging, "json-logging", false, "Always use JSON logging")
 
+	var zoneSyncEnforceIntervalDNSController int
+	var syncIntervalDNSController int
+	flag.IntVar(&zoneSyncEnforceIntervalDNSController, "zone-sync-interval", 45, "DNSController frequency of retrieving current zones from dns provider")
+	flag.IntVar(&syncIntervalDNSController, "zone-sync-interval", 3, "DNSController frequency of uploading to dns provider on change.")
```

Comment:
Please add the unit. Is this ms? seconds? minutes?

Reply:
Also, are these decent default values? MB we should increase them

Reply:
I agree. went with the old values